lib/core/stores/remote.ts

```diff
@@ -289,4 +392,9 @@ export class RemotePromiseStore implements IPromiseStore {
       throw new ResonateError(ErrorCodes.ENCODER, "Decode error", e);
     }
   }
+
+  // any response from delete is true
+  private isDeletedResponse(_: any): _ is true {
```

Comment:
```suggestion
  private isDeletedResponse(_: any): _ is any {
```

We could even just use an anonymous function since this is a one liner